tsconfig.json

```diff
@@ -1,6 +1,13 @@
 {
   "compilerOptions": {
     "target": "ES2020",
+    "module": "ESNext",
+    "lib": [
+      "ESNext",
+      "ES2022",
```

Comment:
I don't _think_ you'll need ES2022 if you also have ESNext. ESNext should be a superset of ES2022.